test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -404,6 +414,23 @@ public Task StopAsync()
             }
         }
 
+        private static bool IsWebSocketsSupported()
```

Comment:
Lets move this to `Microsoft.AspNetCore.SignalR.Tests.Utils` and reuse it everywhere (including that conditional test attribute)

Reply:
Yeah, probably about time to move this

Reply:
In fact I can see a `TestHelpers.IsWebSocketsSupported()` method already exists in that project. Check with @anurse to see whether this logic is better than what is already in that method and possibly replace it.

Reply:
Yeah, I wen't ahead and replaced the logic. I can reverted it though. Thoughts? @davidfowl  @anurse

Reply:
Remove

Reply:
I think the new logic is good. This copy isn't needed though right?